<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencedialog</class>
 <widget class="QDialog" name="preferencedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="calTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Calibration &amp;Files</string>
       </attribute>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>441</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>261</height>
         </rect>
        </property>
        <property name="text">
         <string>In this tab you can either add or remove calibrations files from the calibration file-list by simply clicking the &quot;Add File&quot;- or &quot;Remove File&quot;-button. 
After you are finished you can either save the list into a standard file named &quot;xs.cal&quot; by pressing the &quot;Save&quot;-button or you can ignore the changes by pressing the &quot;cancel&quot;-button.
If there was already a calibration file-list saved in the running directory the box on the right side won't be empty. 
It doesn't make any differences if you change the &quot;xs.cal&quot;-file via this preference dialog or a via another editor program.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>311</y>
          <width>291</width>
          <height>62</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>&amp;Add File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>&amp;Remove File</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="detTab">
       <attribute name="title">
        <string>&amp;Detector Settings</string>
       </attribute>
       <widget class="QTableWidget" name="detTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>751</width>
          <height>371</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>751</width>
          <height>371</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>751</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="rowCount">
         <number>35</number>
        </property>
        <property name="columnCount">
         <number>9</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>94</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>DGF Module</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Crate</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Slot</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Subadress</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Calibration</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Detector Type</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rate limit</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>&amp;Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
